cmake/modules/hunter_autotools_project.cmake

```diff
@@ -315,11 +314,6 @@ function(hunter_autotools_project target_name)
       "PATH=${PARAM_GLOBAL_INSTALL_DIR}/bin:${default_path}"
   )
 
-  # PKG_CONFIG_PATH environment variable
```

Comment:
`hunter_finalize` will not be called in scheme unless there is `hunter_add_package` (i.e. dependency). I think this should be kept.